tools/SourceKit/CMakeLists.txt

```diff
@@ -234,6 +238,11 @@ macro(add_sourcekit_executable name)
       set_target_properties(${name}
         PROPERTIES
         LINK_FLAGS "-Wl,-exported_symbol,_main")
+
+      if(SWIFT_ANALYZE_CODE_COVERAGE)
+        set_property(TARGET "${name}" APPEND_STRING PROPERTY
```

Comment:
`_add_variant_c_compile_link_flags` does return the two code coverage flags, but they're overwritten later by `set_target_properties`. Building the project without these snippets results in linker errors due to missing code coverage libraries.